### DISCUSSION TITLE
DFR0995 (DFRobot 1.47" 172x320px LCD)

### DISCUSSION THREAD

**Discussion by EdgarKluge** [peterhinch/micropython-nano-gui#55] [Ideas]
Hi Peter,

I have adapted your driver (st7789_4bit.py) for the display 1.47" 172×320 IPS LCD Display Module from DFRobot.
I had to add the following display type:

DFR0995 = (34, 0, 0) # DFR0995

One more display to work with nano-gui.
Thanks for the excellent work & best regards,
Edgar

**Comment by peterhinch** [comment]
Thank you. Now updated.
